application-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -110,13 +119,19 @@
           #if ($element == 'body')
             #set ($element = '')
           #end
+          #if ($element == '')
+            #set ($orphan = true)
+          #else
+            #set ($orphan = false)
+          #end
```

Comment:
I think you can write `#set ($orphan = ($element == ''))` or `#set ($orphan = $stringtool.isBlank($element))` (better)